geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -56,6 +56,8 @@ class AccumuloDataStoreTest extends Specification with TestWithMultipleSfts {
 
   val defaultGeom = WKTUtils.read("POINT(45.0 49.0)")
 
+  val keywordsDelimitter = "|"
```

Comment:
I'd suggest moving this to geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala.